src/traces/scattermapbox/attributes.js

```diff
@@ -26,6 +26,61 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            role: 'info',
+            dflt: false,
+            editType: 'plot',
+            description: 'Determines whether clustering is enabled or disabled.'
+        },
+        maxzoom: {
+            valType: 'number',
+            role: 'info',
+            min: 0,
+            max: 22,
+            dflt: 14,
+            description: 'Sets the maximum zoom level for the cluster.',
+        },
+        radius: {
+            role: 'info',
+            valType: 'number',
+            dflt: 50,
+            description: 'Radius of each cluster when clustering points.',
+        },
+        steps: {
+            role: 'info',
+            valType: 'number',
+            arrayOk: true,
+            dflt: -1,
+            min: -1,
+            description: [
+                'Sets the steps for each cluster.'
```

Comment:
I'm confused about what this attribute does?

Reply:
The cluster can be coloured in different colors and take different sizes dependent on how many observations that are contained in a cluster. The steps determines when the cluster will change size or/and color

Reply:
I didn't notice any changes when trying to tweak this parameter.
@elben10 wondering if you could provide an illustration?

Reply:
> The cluster can be coloured in different colors and take different sizes dependent on how many observations that are contained in a cluster. The steps determines when the cluster will change size or/and color

Thanks @elben10 please provide part of this info in the attribute description.